website/versioned_docs/version-0.2.x/first-composite.mdx

```diff
@@ -9,6 +9,12 @@ This page presents how to create your first composite and deploy it to your loca
 
 ## Running a local Ceramic node
 
+:::caution
+
+Because ComposeDB is still an experimental feature set built on top of Ceramic, if you want to use it with a Ceramic node, you need to set the `CERAMIC_ENABLE_EXPERIMENTAL_INDEXING` environment variable to `true`, before running a node.
```

Comment:
I tried to play with composeDB this morning and saw this pr, after following the first composite guide I tried to query the test model, and hit the `Error: Index for historical data on kjzl6hvfrbw6ca7nidsnrv78x7r4xt0xki71nvwe4j5a3s9wgou8yu3aj8cz38e is not available` error. I don't know how big is the data to be imported but might be nice to have a note when we add the model to the `daemon.config.json` file to enable this option :).

Reply:
Hey, @pradel !

Thanks for sharing! Can you share what steps exactly you followed to get the `Error: Index for historical data on kjzl6hvfrbw6ca7nidsnrv78x7r4xt0xki71nvwe4j5a3s9wgou8yu3aj8cz38e is not available` error?

ComposeDB is not released officially yet. We're preparing for a Developer Preview release in a couple of weeks and this documentation here is gonna be a part of the release.

Reply:
Sure, after the last step, I opened graphiql and ran the following query
```
{
  simpleProfileIndex(first: 20) {
    edges {
      node {
        id
        displayName
      }
    }
  }
}
```
Then you should see the error.

Reply:
Thanks! Will have a look at it and get back to you :).

Reply:
Ok, @pradel , after testing this myself, I think that the only way for you to get the `Error: Index for historical data on kjzl6hvfrbw6ca7nidsnrv78x7r4xt0xki71nvwe4j5a3s9wgou8yu3aj8cz38e is not available` error, is if you have the `allow-queries-before-historical-sync` flag set to `false` in your ceramic node's index config.

This flag is set to `true` by default in the current version of ceramic, but I added an explicit mention to the docs that it needs to be set to `true`. Does it work for you :) ?

Reply:
@pawartur looks like it fixed the error 👍. Maybe it's because I was running a previous version of the daemon before but this setting was not set at all in my config. Now the query is working and does not throw any error, the result of the query is an empty array so not sure if it's the normal behavior or if some data should be present.

Reply:
Sweet! Thanks for getting back to me, @pradel :).

Yes, I think that must have been because you had had an old config file created while running an older version of the daemon.

Yes, it's normal at this stage that you're getting empty results. The results are returned from the node's index and currently the index will only be populated with data appearing on the network while the node is running. We are working on implementing "historical data sync" which will populate the index with all the data for indexed models.

For now, if you want to see some data in your index, the easiest way is to create the data yourself using `ComposeClient`. But if you have your node indexing the example model, connected to clay and running constantly, you may also see instances of the model created by other devs testing ComposeDB :).

Reply:
Amazing, thanks for the explanation :)
Will continue to play with compose over the next few days!

Reply:
Thank You, @pradel ! Please, let us know, if you have any more feedback :).